### PR TITLE
Fixed task label tutorial for new UI

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: 1.8/usage/tutorials/task-labels.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Assigning Labels to Applications and Tasks
+
+You can attach labels to tasks either via the **Services** tab of the DC/OS web interface or from the DC/OS CLI using the Marathon HTTP API.
+
+From the DC/OS web interface, click the **Services** tab. You can add labels when you deploy a new service or edit an existing one from the **Labels** tab. When you deploy the application, the label name and value will be associated with it. You can query the label using the Marathon HTTP API. You can specify more than one label, but each label can have only one value.
+
+You can also specify label values in the `labels` parameter of your application definition. 
+
+    $ vi myap.json
```
Comment (joel-hamill) [inline 1.8/usage/tutorials/task-labels.md]: `vi myapp.json`?

File: 1.8/usage/tutorials/task-labels.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+menu_order: 12
+---
+
+This tutorial illustrates how labels can be defined using the DC/OS web interface and the Marathon HTTP API, and how information pertaining to applications that are running can be queried based on label value criteria.
+
+When you deploy applications or containers in a DC/OS cluster, you can associate a tag or label with your deployed components in order to track and report usage of the cluster by those components. For example, you may want to assign a cost center identifier or a customer number to a Mesos application and produce a summary report at the end of the month with usage metrics such as the amount of CPU and memory allocated to the applications by cost center or customer.
+
+## Assigning Labels to Applications and Tasks
+
+You can attach labels to tasks either via the **Services** tab of the DC/OS web interface or from the DC/OS CLI using the Marathon HTTP API.
```
Comment (joel-hamill) [inline 1.8/usage/tutorials/task-labels.md]: - using the Marathon HTTP API -

File: 1.8/usage/tutorials/task-labels.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+From the DC/OS web interface, click the **Services** tab. You can add labels when you deploy a new service or edit an existing one from the **Labels** tab. When you deploy the application, the label name and value will be associated with it. You can query the label using the Marathon HTTP API. You can specify more than one label, but each label can have only one value.
+
```
Comment (joel-hamill) [inline 1.8/usage/tutorials/task-labels.md]: From these instructions it's unclear that you are switching between UI and CLI instructions. Maybe add a heading for UI and CLI, similar to https://docs.mesosphere.com/1.8/usage/managing-services/install/.

File: 1.8/usage/tutorials/task-labels.md (hunk 2 above)
Comment (joel-hamill) [inline 1.8/usage/tutorials/task-labels.md]: I think we can delete `using the Marathon HTTP API` since this is an implementation detail at this point.

**Comment by sascala** [comment]
Thanks for your comments, @joel-hamill

**Pull request merged**